.github/workflows/publish.js.yml

```diff
@@ -21,6 +21,7 @@ jobs:
     - run: npm install
       name: Install dependencies
     - run: rm -rf package-lock.json
+    - run: rm -rf node_modules/appium
     - run: npm shrinkwrap
```

Comment:
do we shrinkwrap dev dependencies as well?

Reply:
perhaps we should apply something like https://github.com/npm/npm/issues/11189

Reply:
Maybe:

1. run npm run test first after `npm install`
2. remove all node_modules and runs `npm install --production`
3. create `shrinkwrap`

can do. One possible concern is modules installed in step2) could be different from 1)'s npm run one (e.g. a new module version has been released between them).

Reply:
The current change should be ok.